### ISSUE TITLE
Putting in video links

### ISSUE THREAD

**Issue by christhron** [jthomasbarry/aafmt#43]
I can see the command in the Latex code that links to video, but I don't understand how it works.  The command I see in permutations.text is:

\sectionvideohref{KJyItH3ZgpU&list=PL2uooHqQ6T7PW5na4EX8rQX2WvBBdM8Qo&index=30}}

I don't understand how to form what's inside the {}.  It's not the same as the web address. If you can explain this to me, I can get a student to put in the video links.

I also noticed that the video icon in Section 10.6 no longer works.

Thanks

**Comment by davidfarmer** [comment]
Isn't the string in brackets the part of the YouTube URL
that comes after the "v="?

We could change the macro so that you give it the entire URL
to the video, if that would work better for you.

Somehow the video_camera.png file was deleted from the
images directory.  I'll email it to you and you can put it
back.

**Comment by christhron** [comment]
Hi David,

The video_camera.png image is already there, I checked. Did it get lost on
your end?

As to the youtube, I guess that was the long version of the address. I will
try and see if I can make it work.

Best wishes
chris

**Comment by davidfarmer** [comment]
I thought that image was not in the GitHub version, but apparently
I was mistaken.

The html version of the video link is still working.  I noticed that
the linked URL only used part of the long string in the sectionvideohref
command.  Not sure if that matters.

I just pulled, recompiled with pdflatex, and the video link worked just 
fine.  The only thing I can think of is that it opened in a browser
window which you did not see.

I'll recompile the html.
